code/modules/admin/DB_ban/functions.dm

```diff
@@ -390,7 +390,30 @@
 	output += "Please note that all jobban bans or unbans are in-effect the following round."
 
 	if(adminckey || playerckey)
-
+		playerckey = sanitizeSQL(ckey(playerckey))
+		adminckey = sanitizeSQL(ckey(adminckey))
+		var/playersearch = ""
+		var/adminsearch = ""
+		if(playerckey)
+			playersearch = "AND ckey = '[playerckey]' "
+		if(adminckey)
+			adminsearch = "AND a_ckey = '[adminckey]' "
+		var/bancount = 0
+		var/bansperpage = 15
+		var/pagecount = 0
+		page = text2num(page)
+		var/DBQuery/query_count_bans = dbcon.NewQuery("SELECT COUNT(id) FROM [format_table_name("ban")] WHERE 1 [playersearch] [adminsearch]")

```

Comment:
Only in case, when ``playersearch`` or ``adminsearch`` exist. 
You should try pass " " param (whitespace), it will be true on 392, but after ``ckey()`` it will most likely false.